src/rust/ide/view/graph-editor/src/component/node/action_bar.rs

```diff
@@ -0,0 +1,241 @@
+//! Definition of the `ActionBar` component for the `visualization::Container`.
+
+use crate::prelude::*;
+
+use crate::component::node;
+
+use enso_frp as frp;
+use enso_frp;
```

Comment:
```suggestion
use enso_frp as frp;
```
And replace enso_frp usages with frp.